src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (num) => {
```

Comment:
```suggestion
  let sum = 0;
  
  const adder = (num) => {
```
it's better to separate variables declaration and functions declaration just for better readability